invenio_circulation/records/loaders/schemas/json.py

```diff
@@ -120,6 +120,17 @@ def validate_transaction_user_pid(self, value, **kwargs):
                 field_names=["transaction_user_pid"],
             )
 
+    @post_load
+    def postload_fields(self, data, **kwargs):
+        """Post-load record fields update."""
+        record = self.context.get("record")
+        if record:
+            new_record = dict(record)
```

Comment:
If I am not wrong this is a shallow copy, maybe just doing:
```
record.update(data)
return record
```
is doing the same? WDYT?